### DISCUSSION TITLE
Full support for web version of Code

### DISCUSSION THREAD

**Discussion by mikehudson2** [PDConSec/vsc-print#219] [Ideas]
Printing from Github Codespaces, i.e. from within a browser, appears only in black and white with a plain font. Is this the issue referred to in 'troubleshooting' as "At the time of this release, problems with command routing were causing printing from remote workspaces to fall back to using the default printer. Full service will be restored as soon as possible. " ?

**Comment by PeterWone** [comment]
Are you telling me it is possible to use this extension with the web version of the VS Code? If so, I didn't know that.

The issue relating to command routing is another matter. There are remoting extensions for VS Code that will allow you to open a workspace that's in a docker container or on a remote Unix machine via ssh. Under those conditions there is a problem with command routing that prevents the remote proxy (which runs on the Unix host) from triggering the alternate browser on the workstation.

**Comment by mikehudson2** [comment]
Yes, it does work on the web version but only in B+W with a plain font. Needless to say it would be great if you are able to get the formatting sorted for the web version too.

**Comment by PeterWone** [comment]
I will look into it. But not soon, I fear. My time is already overcommitted. 

I appreciate the feedback. Also, if you have the skills and the time, I'm always open to contributions as PRs.

**Comment by mikehudson2** [comment]
Hi Peter. Has there been any progress in getting this extension to work on the web version of Codespaces? I understand you are super busy but it never hurts to ask! Cheers, Mike
